docs/DeveloperGuide.md

```diff
@@ -257,14 +257,14 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* Professor in charge of CS2103
+* Professor in from NUS School of Computing
```

Comment:
Perhaps a minor error here - an extra "in"

Reply:
Good catch!